hdr_schemata/models/HDRUK/v2_1_3/Temporal.py

```diff
@@ -0,0 +1,16 @@
+from pydantic import Field
+from hdr_schemata.definitions.HDRUK import Periodicity
+from hdr_schemata.models.HDRUK.base.Temporal import Temporal as BaseTemporal
```

Comment:
I'm wondering if it would make sense to import `Temporal` from the latest schema version rather than from `HDRUK.base`? 

Hypothetically, say for version `2.1.4` we wanted to change the name `timeLag` to `timeDelay` or something.  For `2.1.4` the developer would import `base.Temporal` and have to make two modifications to it (`timeLag` -> `timeDelay` and `accrualPeriodicity` -> `publishFrequency`).  I imagine it would be easy to miss a previous iteration that was made to `base.Temporal`.  IIUC, importing from the latest version by default should mean the developer doesn't have to go back and check previous iterations.

Reply:
Great that you checked but this is simply because `2.1.2` is the base and the latest version.

While we have older versions of the schema, these are json-schema only and not pydantic.  Therefore the 'base' of the HDRUK schema is `2.1.2`. When we make a `2.1.4` then yeh definitely you're right to import from the latest schema which would be `from hdr_schemata.models.HDRUK.v2_1_2.Temporal`

Reply:
Could rename `Base` to be `2.1.2` if that's less confusing?

Reply:
Right of course, I hadn't realised this was a special case because `2.1.2` is the `Base`.  It's probably worth writing some documentation about how to make these updates to the schema re importing from the latest version.